scripts/ft-benchmark.sh

```diff
@@ -8,6 +8,7 @@ date
 # Otherwise nearup and cargo don't work even if installed properly
 PATH=/home/ubuntu/.local/bin/:$PATH
 export PATH=$PATH:$HOME/.cargo/bin
```

Comment:
Preexisting, but a good opportunity to:

```suggestion
PATH="$HOME/.local/bin/":"$PATH"
export PATH="$PATH":"$HOME/.cargo/bin"
```